### DISCUSSION TITLE
Copy unidoc API documentation into Laika

### DISCUSSION THREAD

**Discussion by sbrunk** [planet42/Laika#361] [Q&A] [answered]
I'm using the [sbt-unidoc](https://github.com/sbt/sbt-unidoc) project to combine Scaladoc from multiple modules. sbt-unidoc merges the Scaladoc of all modules and puts it into `target/unidoc` in the project where the unidoc plugin is enabled.

I'm looking for a way to configure the input of the `laikaGenerateAPI` task or some other way to include the output of sbt-unidoc instead of regular Scaladoc. Does anyone know how this could be achieved?

**Comment by armanbilge** [comment, marked as answer]
I think you should override the value of this setting
https://github.com/planet42/Laika/blob/135eceab026e10550d660699268390e4c43a349b/sbt/src/main/scala/laika/sbt/LaikaPlugin.scala#L171
to
```scala
(unidocProject / ScalaUnidoc / packageDoc / mappings).value
```

**Reply by sbrunk** [reply]
That's working fine! Thanks @armanbilge
